src/Components/DisplayBook.js

```diff
@@ -4,11 +4,11 @@ import InputBooks from './InputBooks';
 
 const DisplayBooks = () => {
   const books = useSelector((state) => state.books.books);
-
   return (
     <>
       <BookList books={books} />
       <InputBooks />
+      <footer>Made with ❤ by Bilal Ahmed</footer>
```

Comment:
- [optional] please kindly add some padding top and bottom to the footer, so the input fields can be more clear. Nice work.